src/auth.rs

```diff
@@ -361,6 +369,14 @@ pub async fn verify_identity(pubkey: &str, keyserver: &str, account: &str) -> Re
         Err(AuthError::CacaoAccountMismatch)?;
     }
 
+    if let Some(resources) = cacao.p.resources {
+        if !resources.contains(&iss.to_owned()) {
+            Err(AuthError::CacaoResourcesMissingIss)?;
```

Comment:
Need a log statement here?

Reply:
wdym? It's an error which is logged up the stack